Expecto.Tests/Tests.fs

```diff
@@ -181,6 +181,9 @@ let tests =
 
 [<Tests>]
 let expecto =
+
+  let testArgs i o = Expect.equal (Args.parseOptions options i) o "testArgs"
```

Comment:
Consider `sequence equal`

Reply:
It's comparing a `Result<args list,string list>` so a bit more tricky.